_api-reference/document-apis/get-documents.md

```diff
@@ -11,27 +11,87 @@ redirect_from:
 **Introduced 1.0**
 {: .label .label-purple }
 
-After adding a JSON document to your index, you can use the get document API operation to retrieve the document's information and data.
+After adding a JSON document to your index, you can use the Get Document API operation to retrieve the document's information and data.
 
-## Example
+
+## Realtime
+
+The Get Document API in OpenSearch operates in realtime mode by default, meaning it retrieves the latest version of the document regardless of the index's refresh rate (which determines when new data becomes searchable). However, if you request stored fields (using the `stored_fields` parameter) for a document that has been updated but not yet refreshed, the Get Document API needs to parse and analyze the document's source to extract those stored fields. To disable this realtime behavior and retrieve the document based on the last refreshed state of the index, set the `realtime` parameter to false.
+
+## Source filtering
+
+By default, the Get Document API returns the entire contents of the `_source` field for the requested document. However, you can choose to exclude the `_source` field from the response by using the `_source` URL parameter and setting it to false, as shown in the following example:
 
 ```json
-GET sample-index1/_doc/1
+GET test-index/_doc/0?_source=false
 ```
-{% include copy-curl.html %}
+
+### `source` includes and excludes
+
+If you only need specific fields from the `_source`, use the `_source_includes` or `_source_excludes` parameters to include or exclude particular fields, respectively. This can be beneficial for large documents, as retrieving only the required fields can reduce network overhead. Both parameters accept a comma-separated list of fields or wildcard expressions, as shown in the following example:
+
+```
+GET test-index/_doc/0?_source_includes=*.id&_source_excludes=entities
```

Comment:
Can we check if this is this a correct example ? 

if '_source_includes' only contains '*.id' , implicitly we are excluding anything else including 'entities' ?